content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -314,6 +314,16 @@ on:
     types: [created, deleted]
 ```
 
+The `issue_comment` event occurs for both, comments on issues as well as pull requests. To run jobs only in one of the cases, you can check the event payload for the `issue.pull_request` attribute and use it as condition to skip jobs.
+
+```yaml
+on: issue_comment
+
+jobs:
+  pr_commented:
+    if: ${{ github.event.issue.pull_request }}
```

Comment:
What do you think about adding the opposite case too?

```suggestion
    name: New PR comment
    if: ${{ github.event.issue.pull_request }}
    runs-on: ubuntu-latest
  
    steps:
      - run: echo "New PR comment"
  
  // This job only runs when an issue comment is created
  issue-commented:
    name: New issue comment
    if: ${{ !github.event.issue.pull_request }}
    runs-on: ubuntu-latest
  
    steps:
      -  run: echo "New issue comment"
```

Reply:
Yeah, why not.

The comment should start with `#` not `//`

Reply:
This job only runs when an issue **comment is created** - this isn't actually true for the example. It will trigger for `created`, `edited` and `deleted` (default). I changed the comment to avoid activity types altogether in this example.

Reply:
Good catch @Simran-B! 🙌